lib/simplecov/combiners/branches_combiner.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SimpleCov
+  module Combiners
+    #
+    # Combine different branche coverage results on single file
+    #
+    class BranchesCombiner < BaseCombiner
+      #
+      # Return merged branches or the existed branche if other is missing
+      #
+      # @return [Hash]
+      #
+      def combine
+        return existed_coverage unless empty_coverage?
+        combine_branches
+      end
+
+      #
+      # Branches inside files are always same if they exists, the difference only in coverage count
+      # Branch coverage report for any conditional case is build from hash it's key is condition and
```

Comment:
`is built` vs `is build`. Also, a comma or a dot should be place before `it's key` because those are different sentences.

Reply:
fixed